portfolio.sql

```diff
@@ -0,0 +1,104 @@
+/*!40014 SET @OLD_UNIQUE_CHECKS=@@UNIQUE_CHECKS, UNIQUE_CHECKS=0 */;
+/*!40014 SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0 */;
+/*!40101 SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
+/*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;
+
+--
+-- Table structure for table `about_me`
+--
+
+DROP TABLE IF EXISTS `about_me`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `about_me` (
+  `id` int(11) unsigned NOT NULL AUTO_INCREMENT,
+  `paragraph_1` text,
+  `image` text,
+  `paragraph_2` text,
+  PRIMARY KEY (`id`)
+) ENGINE=InnoDB AUTO_INCREMENT=2 DEFAULT CHARSET=utf8;
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+--
+-- Dumping data for table `about_me`
+--
+
+LOCK TABLES `about_me` WRITE;
+/*!40000 ALTER TABLE `about_me` DISABLE KEYS */;
+INSERT INTO `about_me` VALUES (1,'I am a junior web developer with a passion for design and agile development. I am currently learning HTML, CSS, Javascript, PHP, and MySQL. I am also familiar with Git and other software environments. I am a certified scrum master and will be applying for my PHP Zend certification after I have graduated from Mayden Academy.','img/a-drop-of-water-osa-water-leaf-158725.jpeg','In addition to my web-development skills I have an honours degree in marketing and have over ten year\'s experience in hospitality and customer service. I also possess a level 5 leadership qualification from the Institute of Business Leadership. As far as hobbies and personal time go, I spent a lot of time working on my car, keeping fit, learning German and playing Rugby for the Bath Saracens. I am also currently working part-time at the King William pub in Bath.');
+/*!40000 ALTER TABLE `about_me` ENABLE KEYS */;
+UNLOCK TABLES;
+
+--
+-- Table structure for table `intro`
+--
+
+DROP TABLE IF EXISTS `intro`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `intro` (
+  `id` int(11) unsigned NOT NULL AUTO_INCREMENT,
+  `title` varchar(255) DEFAULT NULL,
+  `intro_text` text,
+  PRIMARY KEY (`id`)
+) ENGINE=InnoDB AUTO_INCREMENT=2 DEFAULT CHARSET=utf8;
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+--
+-- Dumping data for table `intro`
+--
+
+LOCK TABLES `intro` WRITE;
+/*!40000 ALTER TABLE `intro` DISABLE KEYS */;
+INSERT INTO `intro` VALUES (1,'WEB DEVELOPER','Welcome to my webpage! Please feel free to stay as long as you like and don\'t hesitate to use my contact details at the bottom of the webpage if you would like to get in touch.');
+/*!40000 ALTER TABLE `intro` ENABLE KEYS */;
+UNLOCK TABLES;
+
+--
+-- Table structure for table `projects`
+--
+
+DROP TABLE IF EXISTS `projects`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `projects` (
+  `id` int(11) NOT NULL,
```

Comment:
this doesnt auto increment, this will cause problems

Reply:
rectified in database and re-committed the rectified file.